modules/blocks.php

```diff
@@ -60,6 +66,50 @@ function jetpack_tiled_gallery_load_block_assets( $attr, $content ) {
 	}
 }
 
+/**
+ * Gif block registration/dependency declaration.
+ *
+ * @param array $attr - Array containing the map block attributes.
+ *
+ * @return string
+ */
+function jetpack_gif_block_render( $attr ) {
+	$align       = isset( $attr['align'] ) ? $attr['align'] : 'center';
+	$padding_top = isset( $attr['paddingTop'] ) ? $attr['paddingTop'] : 0;
+	$style       = 'padding-top:' . $padding_top;
+	$giphy_url   = isset( $attr['giphyUrl'] ) ? $attr['giphyUrl'] : null;
+	$search_text = isset( $attr['searchText'] ) ? $attr['searchText'] : '';
+	$caption     = isset( $attr['caption'] ) ? $attr['caption'] : null;
+
+	if ( ! $giphy_url ) {
```

Comment:
It seems that we _require_ both of these to correctly render the block:

```suggestion
	if ( ! $giphy_url || ! $padding_top ) {
```